lib/vdsm/tool/configurators/multipath.py

```diff
@@ -327,44 +327,60 @@ def configure():
     commands.run([_MULTIPATHD.cmd, "reconfigure"])
 
 
-def isconfigured():
+def _check_conf_rev(mpathconf):
     """
-    Check the multipath daemon configuration. The configuration file
-    /etc/multipath.conf should contain a tag in form
+    The multipath configuration file should contain a tag in form
     "RHEV REVISION X.Y" for this check to succeed.
     If the tag above is followed by tag "RHEV PRIVATE" the configuration
     should be preserved at all cost.
+
+    Args:
+        mpathconf (List[str]): Lines in the multipath daemon
+        configuration file.
+
+    Returns:
+        bool: check has passed.
+    """
+    first = mpathconf[0]
+    second = mpathconf[1]
```

Comment:
Letting IndexError propagate here is not great. We don't want that the caller will have to know that we raise IndexError, and this can hide real error in other part of the function.

The error should be handled here, and we need to log a good error and return the right value to the caller.

Reply:
Uhm, afterwards I change it for `readline()` that avoids the exception. I'll reorder the patches so that it happens first, and we don't need to worry about the exception at all.